OsmAnd/res/values/strings.xml

```diff
@@ -289,9 +289,9 @@
     <string name="quick_action_add_destination">Add destination</string>
     <string name="quick_action_replace_destination">Replace destination</string>
     <string name="quick_action_add_first_intermediate">Add first intermediate</string>
-    <string name="quick_action_add_destination_desc">Tap the action button to make your previous destination into an intermediary waypoint to a new destination in the centre of the screen.</string>
-    <string name="quick_action_replace_destination_desc">Center the view away from the destination and onto yourself by tapping the action button.</string>
-    <string name="quick_action_add_first_intermediate_desc">Add an intermediate waypoint before your destination in the middle of the screen by tapping the action button.</string>
+    <string name="quick_action_add_destination_desc">Tap the action button to add a destination at the screen center location. The previously set destination becomes the last intermediate destination.</string>
+    <string name="quick_action_replace_destination_desc">Tap the action button to replace the destination with the screen center location.</string>
```

Comment:
"Tap the action button" is not the info that is conveyed. I see I did the same above, but nontheless. It is hard for me to read this sentence. It sounds a bit like the destination is moved, is that the case?